libs/ui-toolkit/src/utils/intent.tsx

```diff
@@ -28,7 +28,7 @@ export const getIntentBorder = (intent = Intent.None) => {
   };
 };
 
-export const getIntentTextAndBackground = (intent = Intent.None) => {
+export const getIntentTextAndBackground = (intent?: Intent) => {
```

Comment:
because there's no lozenge component available which would suit the designs i need. if it defaults to Intent.NONE, i can't override the classes even when i inject a className string to the component, which is super weird. this was one of the easy ways to free up the component and make it more dynamic to suit the occasional custom stuff, which we (as it turns out) have to deal with. we probably wouldn't need this if the designs would never contain custom versions.

Reply:
Seems reasonable for now